server/src/pages/shot/view.js

```diff
@@ -452,6 +456,24 @@ class Body extends React.Component {
     </reactruntime.BodyTemplate>);
   }
 
+  promoOpen(promoType) {
+    if (promoType === "edit") {
```

Comment:
We don't use the promo card for anything else, so get rid of `promoType` and associated conditionals.